src/main/java/com/flightstats/hub/cluster/DistributedLockRunner.java

```diff
@@ -0,0 +1,39 @@
+package com.flightstats.hub.cluster;
+
+import lombok.extern.slf4j.Slf4j;
+
+import javax.inject.Inject;
+import java.util.concurrent.TimeUnit;
+import java.util.function.Consumer;
+
+/**
+ * DistributedLockRunner is intended for short running processes, since it is blocking.
+ */
+@Slf4j
+public class DistributedLockRunner {
```

Comment:
same here. Why is it LockRunner? Why not just say DistibutedLock?

Reply:
I was thinking that it's the class that runs a thing in a lock, not the lock itself...